### PR TITLE
Fixes StretchyView bug on overscroll effect

### PR DESCRIPTION
**Pull request by burkeshartsis**
### What does this PR do, or why is it needed?
This PR fixes a bug in StretchyView that was causing a crash on iOS. It does this by:
- correcting an api syntax error
- using `Animated.ScrollView`

Coincidentally, this might actually improve `StretchyView` performance on iOS by actually passing its animation to the native thread, where it might not have been doing so previously.

### What design trade-offs/decisions were made?
Additionally, this PR adds a commit (92c0151), **that you are welcome to keep or revert**, that removes all `StretchyView` functionality from Android. It basically just renders a `View` with no additionally renders or computational logic. This is because `StretchyView` doesn't actually work on Android to begin with because Android doesn't overscroll. No overscroll, no way to animate the image with a stretchy overscroll effect. So might as well ditch all the code right 🤷‍♂ Like I said, keep it or revert it. I leave that decision to the reviewer.

### How do I test this PR?
- open app
- tap on an Article/Sermon/Series (anything but a Devo single)
- everything look correct?
- pull down to see the overscroll stretchy effect on ios and scroll on android.
- No error/crash? We're good!

### What automated tests did you write?
All the changes in this PR are unable to be covered with jest snapshots.

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-kit/src/StretchyView/index.android.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <View style={[background ? StyleSheet.absoluteFill : null, style]}>
+      {children}
+    </View>
+  );
+
+  render() {
+    return (
+      <View style={StyleSheet.absoluteFill}>
+        {this.props.children({
+          Stretchy: this.StretchyPortal,
```
Comment (vinnyjth) [inline packages/apollos-ui-kit/src/StretchyView/index.android.js]: We might want to pass the same arguments down as the ios version, but with empty or `noop`, just in case a client decides to reference each argument individually instead of spreading the arguments.

Comment (burkeshartsis) [reply packages/apollos-ui-kit/src/StretchyView/index.android.js]: fixed in db6cba7
